src/components/common/Layout/Layout.tsx

```diff
@@ -1,15 +1,30 @@
+import NextHead from 'next/head'
+
+import {constants} from '@lib/common'
+
 interface LayoutProps {
   navigation?: React.ReactNode
   content: React.ReactNode
   footer?: React.ReactNode
 }
 
 const Layout: React.FC<LayoutProps> = ({navigation, content, footer}) => (
-  <div className="container flex flex-col min-h-screen mx-auto">
-    {navigation}
-    <main className="flex flex-col flex-1">{content}</main>
-    {footer}
-  </div>
+  <>
+    <NextHead>
+      {constants.site.cloudflareAnalyticsId && (
+        <script
+          defer
+          src="https://static.cloudflareinsights.com/beacon.min.js"
+          data-cf-beacon={`{"token": "${constants.site.cloudflareAnalyticsId}"}`}
+        ></script>
+      )}
+    </NextHead>
```

Comment:
I think it's better to have this in the `_app.tsx`.

Reply:
:thinking: Thanks for letting me know of this, we actually have a `Head` component already that's being used in `_app.tsx` and `_document.tsx`, would it be better to put it under there?

Reply:
Yes! That works, thanks